### DISCUSSION TITLE
Ubuntu22 - fatal server error, xfce4/Gtk warnings, cannot open display

### DISCUSSION THREAD

**Discussion by DocMinus** [kasmtech/KasmVNC#168] [Q&A]
Hi, 
maybe someone has some ideas how to fix this? 
Ubuntu 22 (in a VM provided by our IT at work; they only provide the VMs, no support for Linux stuff...) and xfce4.
I have xfce4 installed (and the goodies), but, the server doesn't even startup. Here are some the errors from the .vnc log file.

(xfwm4:13609): xfwm4-WARNING **: 10:02:42.031: Unsupported GL renderer (llvmpipe (LLVM 15.0.7, 256 bits)).
 2023-08-04 10:02:42,032 [DEBUG] Selection: Selection owner change for XFWM4_COMPOSITING_MANAGER
Xvnc: ../../include/privates.h:121: dixGetPrivateAddr: Assertion `key->initialized' failed.
(EE)
(EE) Backtrace:
(EE) 0: /usr/bin/Xvnc (OsLookupColor+0x139) [0x64f209]
(EE) 1: /lib/x86_64-linux-gnu/libc.so.6 (__sigaction+0x50) [0x7fc0ddd1256f]
(EE) 2: /lib/x86_64-linux-gnu/libc.so.6 (pthread_kill+0x12c) [0x7fc0ddd66a7c]
(EE) 3: /lib/x86_64-linux-gnu/libc.so.6 (raise+0x16) [0x7fc0ddd12476]
(EE) 4: /lib/x86_64-linux-gnu/libc.so.6 (abort+0xd3) [0x7fc0ddcf87f3]
(EE) unw_get_proc_name failed: no unwind info found [-10]
(EE) 5: /lib/x86_64-linux-gnu/libc.so.6 (?+0x0) [0x7fc0ddcf870c]
(EE) 6: /lib/x86_64-linux-gnu/libc.so.6 (__assert_fail+0x46) [0x7fc0ddd09e96]
(EE) 7: /usr/bin/Xvnc (vncClientGone+0xe5) [0x46b575]
(EE) 8: /usr/bin/Xvnc (xvnc_sync_dri3_pixmap+0x209) [0x46b999]
(EE) 9: /usr/bin/Xvnc (present_copy_region+0x43) [0x4dea43]
(EE) 10: /usr/bin/Xvnc (present_execute_copy+0x73) [0x4df023]
(EE) 11: /usr/bin/Xvnc (present_set_abort_flip+0x1c7) [0x4e08b7]
(EE) 12: /usr/bin/Xvnc (present_fake_get_ust_msc+0xc7) [0x4df257]
(EE) 13: /usr/bin/Xvnc (TimerSet+0x180) [0x649180]
(EE) 14: /usr/bin/Xvnc (WaitForSomething+0x258) [0x6492a8]
(EE) 15: /usr/bin/Xvnc (Dispatch+0xb2) [0x5fbbc2]
(EE) 16: /usr/bin/Xvnc (dix_main+0x35a) [0x5ffe1a]
(EE) 17: /lib/x86_64-linux-gnu/libc.so.6 (__libc_init_first+0x90) [0x7fc0ddcf9da0]
(EE) 18: /lib/x86_64-linux-gnu/libc.so.6 (__libc_start_main+0x80) [0x7fc0ddcf9e40]
(EE) 19: /usr/bin/Xvnc (_start+0x25) [0x468585]
(EE)
(EE)
Fatal server error:
(EE) Caught signal 6 (Aborted). Server aborting
(EE)

(xfsettingsd:13626): xfsettingsd-ERROR **: 10:02:42.057: Unable to open display.
xfce4-panel: Cannot open display: .
Type "xfce4-panel --help" for usage.

(Thunar:13638): Gtk-WARNING **: 10:02:43.300: cannot open display: :1
Failed to parse arguments: Cannot open display:

(xfce4-clipman:13644): Gtk-WARNING **: 10:02:43.341: cannot open display: :1.0

(xfce4-notifyd:13645): Gtk-WARNING **: 10:02:43.345: cannot open display: :1.0

** (xiccd:13655): CRITICAL **: 10:02:43.354: Can't open display: :1.0
xcb_connection_has_error() returned true
Failure: Module initialization failed

(xfsettingsd:13653): xfsettingsd-ERROR **: 10:02:43.372: Unable to open display.

(nm-applet:13679): Gtk-WARNING **: 10:02:43.401: cannot open display: :1.0

(polkit-gnome-authentication-agent-1:13682): Gtk-WARNING **: 10:02:43.403: cannot open display: :1.0

(xfce4-power-manager:13683): xfce4-power-manager-ERROR **: 10:02:43.432: Unable to open display.

(evolution-alarm-notify:13678): Gtk-WARNING **: 10:02:43.576: cannot open display: :1.0

**Comment by cpujoe** [comment]
Probably related to #172
